CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ if (WIN32)
   option(WITH_BULLET 		"used for collision detection" 					OFF)
   option(WITH_LCM 		"interprocess communications protocol for visualizers, etc" 			OFF)
 else()
-  option(WITH_DIRECTOR	"vtk-based visualization tool and robot user interface"   ON)
+  option(WITH_DIRECTOR	"vtk-based visualization tool and robot user interface"   OFF)
```

Comment:
whoops, sorry. This went in last night in our rush to get the precompiled release out (we wanted director off for mac) and shouldn't have been on this branch. I'll revert it.